WeatherApp/WeatherAppTests/Controllers/WeatherForecastControllerTest.cs

```diff
@@ -0,0 +1,53 @@
+﻿using NSubstitute;
+using NUnit.Framework;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using WeatherApp.WebSite.Controllers;
+using WeatherApp.WebSite.Models;
+using WeatherApp.WebSite.Services;
+
+namespace WeatherApp.Tests.Controllers
+{
+    [TestFixture]
+    class WeatherForecastControllerTest
+    {
+        private IWeatherForecastService _weatherForecastService;
+        private WeatherForecastController _weatherForecastController;
+
+
+        [SetUp]
+        public void Setup()
+        {
+            _weatherForecastService = Substitute.For<IWeatherForecastService>();
+            _weatherForecastController = new WeatherForecastController(_weatherForecastService);
+        }
+
+        [Test]
+        public async Task GetWeatherForecastForCity()
```

Comment:
Be consistent with test naming. -> `Get_ExistingForecast_ReturnForeCast`

Reply:
Thanks for the remark, it's fixed now.